src/target/ext/mod.rs

```diff
@@ -8,6 +8,7 @@ use crate::target::Target;
 
 pub mod breakpoint;
 pub mod monitor;
+pub mod offsets;
```

Comment:
Could we change the module name to `section_offsets`, and the `OffsetsCmd` trait to `SectionOffsets`? That would keep things consistent with the rest of the extensions. 

Ignore `monitor::MonitorCmd`, I've fixed that inconsistency on a local branch lol

Reply:
Alternatively, maybe the names should be changed to something along the lines of `legacy_section_offsets`? 
This would tie-back to the fact that `qOffsets` seems to have been superseded by `qXfer:libraries:read`, and that it would be great to support the new approach as well.

Or possibly even `basic_section_offsets`, reflecting the fact that `qOffsets` only allows specifying text/data relocation.

Up to you really.

Reply:
I've kept the name as `section_offsets` because while there are still modern alternatives, I still think it is probably the easiest method of to handle simple relocation adjustment. It seems unlikely that GDB will deprecate the method any time soon since is the only method supported by QEMU.